events/api/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from events.api import views
+
+urlpatterns = [
+    path('', views.EventsListAPIView().as_view(), name='events'),
```

Comment:
will have to re-check that tomorrow, before merging

Reply:
Better to provide the root events path explicitly:

`path('events', views.EventsListAPIView().as_view(), name='events'),`